deps/graph-parser/test/logseq/graph_parser/util_test.cljs

```diff
@@ -12,3 +12,16 @@
        "4"         false
        "foo bar"   false
        "`property" false))
+
+(deftest extract-file-extension? 
+  (are [x y]
+       (= (gp-util/path->file-ext x) y)
+    "foo.bar" "bar"
+    "foo"     nil
+    "foo.bar.baz" "baz"
+    "../assets/audio.mp3" "mp3"
+    "/root/Documents/audio.mp3" "mp3"
+    "C:\\Users\\foo\\Documents\\audio.mp3" "mp3"
+    "/root/Documents/audio" nil
```

Comment:
Interesting to see we have some nil returns, which was also behavior before this PR. I'll fix up a bug caused by this behavior